src/components/ui/GenericCopyToClipboard/index.js

```diff
@@ -0,0 +1,51 @@
+// @flow
+
+import React, { Component } from 'react'
+import { CopyToClipboard } from 'react-copy-to-clipboard'
+
+import Tooltip from './Tooltip'
+
+import type { Element } from 'react'
+
+const cmz = require('cmz')
+
+const container = cmz(`
+  cursor: pointer
+`)
+
+type Props = {
+  children?: Element<*>
```

Comment:
It's more preferrable to use the `Node` type for the `children` prop, as it includes everything React can render.
```suggestion
  children: Node
```

And don't forget to import `Node` type: `import type { Node } from 'react'`

Check this react Flow definitions for more details: https://github.com/facebook/flow/blob/5958ffb8dd173c86dc5fc9abbfd2a87eea2d4b4a/lib/react.js#L12-L23